brands/amenity/bank.json

```diff
@@ -4701,6 +4701,15 @@
       "name": "State Bank of India"
     }
   },
+  "amenity/bank|State Employees Credit Union": {
+    "tags": {
```

Comment:
Consider adding `"countryCodes": ["us"],` to prevent this credit union from appearing worldwide.